jkube-kit/common/src/main/java/org/eclipse/jkube/kit/common/util/Serialization.java

```diff
@@ -42,7 +42,6 @@ public class Serialization {
   static {
     for (ObjectMapper mapper : new ObjectMapper[]{JSON_MAPPER, YAML_MAPPER}) {
       mapper.enable(SerializationFeature.INDENT_OUTPUT)
-        .enable(SerializationFeature.ORDER_MAP_ENTRIES_BY_KEYS)
```

Comment:
According to 2a478d1f0365d71b01a32ae1a7dcc9c9cf40e2c9 (where this was introduced) it's used to order the Helm `values.yaml` file. That seems like using a sledgehammer to crack a nut. So that was reimplemented in https://github.com/eclipse/jkube/pull/2689/files#diff-33ee7412751801538cf1742ee790e3e2d82ab812d60f3d0c4913a9bf3a433ac0R367.